frontend/compute-system/src/api/urls.ts

```diff
@@ -1,8 +1,9 @@
 const APPSTORE_BASE_URL = "http://localhost:8005";
 export const APPSTORE_URL = `${APPSTORE_BASE_URL}/appstore/`;
 
+export const COCKPIT_BASE_URL = "http://localhost:8006";
+export const COCKPIT_URL = COCKPIT_BASE_URL + '/cockpit'
```

Comment:
use `` `${COCKPIT_BASE_URL}/cockpit` ``

Reply:
there's literally no difference but let it be

Reply:
done